docs/src/static-pages/accessibility.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Developer Accessibility Guidelines
+subtitle: To ensure that we are building the best possible products that work for everyone, we need to consider accessibility at every stage.
+category: guidelines
+---
+
+# Accessible mixins for Web
+
+Within Backpack there are a number of SASS mixins available for use to enable content to be more accessible.
+
+[`bpk-hidden`](https://backpack.github.io/sassdoc/#mixin-bpk-hidden)
+
+Used to hide visually and from screen readers.
+
+[`bpk-visually-hidden`](https://backpack.github.io/sassdoc/#mixin-bpk-visually-hidden)
+
+Hide only visually, but have it available for screen readers: http://snook.ca/archives/html_and_css/hiding-content-for-accessibility
+
+[`bpk-visually-hidden--focusable`](https://backpack.github.io/sassdoc/#mixin-bpk-visually-hidden--focusable)
+
+Modifies the `bpk-visually-hidden` mixin to allow the element to be focusable when navigated to via the keyboard: https://www.drupal.org/node/897638
+
+[`bpk-invisible`](https://backpack.github.io/sassdoc/#mixin-bpk-invisible)
+
+Hide visually and from screen readers, but maintain layout.
+
+# Color
+
+**WHY**
```

Comment:
I don't think any of these headings should be in uppercase but I assume they're copied from that stuff George wrote so I'll give you a pass 😉

Reply:
Yeah copied from elsewhere, was thinking of making them lower but does add more emphasis to these so its clear and assists the reader

Reply:
I'd leave them for now and we can fix later. 

>  add more emphasis to these so its clear and assists the reader

That's what **bold** text and headings are for 😉

Reply:
Resolved!